src/sentry/db/mixin.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import absolute_import
+
+import six
+
+
+from uuid import uuid4
+
+from sentry.models import OrganizationOption
+
+
+def delete_pending_deletion_option(instance, **kwargs):
+    if not issubclass(instance.__class__, PendingDeletionMixin):
```

Comment:
For Python, it's typically much more common to use duck typing for checking here.

So instead of checking the MRO, check `hasattr(instance, 'delete_pending_deletion_option')` or something similar instead. Then it's safe to just say "if this instance has this method, just call it.